### ISSUE TITLE
Crash

### ISSUE THREAD

**Issue by DuckerMan** [csutorasa/XOutput#84]
When I start app I got this:

``` 2019-05-24 15:04:23.474 +03:00 INFO  LanguageManager.set_Language: Language is set to English
2019-05-24 15:04:23.478 +03:00 INFO  MainWindowViewModel.Initialize: Loading settings was successful.
2019-05-24 15:04:23.555 +03:00 ERROR MainWindowViewModel.Initialize: Neither ViGEm nor SCPToolkit is installed.
2019-05-24 15:04:23.568 +03:00 ERROR MainWindowViewModel.UnhandledException: System.NullReferenceException: Object reference not set to an instance of an object.
   at XOutput.UI.Windows.MainWindow.<>c__DisplayClass9_0.<Log>b__0()
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
```

**Comment by ardor97** [comment]
I have the same problem. XOutput desn't work at all. See log:
`2019-05-27 11:05:56.264 +03:00 INFO  LanguageManager.set_Language: Language is set to English
2019-05-27 11:05:56.270 +03:00 INFO  MainWindowViewModel.Initialize: Loading settings was successful.
2019-05-27 11:05:56.282 +03:00 ERROR MainWindowViewModel.Initialize: Neither ViGEm nor SCPToolkit is installed.
2019-05-27 11:05:56.321 +03:00 ERROR MainWindowViewModel.UnhandledException: System.NullReferenceException: Ссылка на объект не указывает на экземпляр объекта.
   в XOutput.UI.Windows.MainWindow.<>c__DisplayClass9_0.<Log>b__0()
   в System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   в System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
`

**Comment by csutorasa** [comment]
I will look into this issue.

**Comment by csutorasa** [comment]
You had no ViGEm installed. I checked without ViGEm and it was not starting up. Fixed the issue in v3.17.1.
